doc/modules/cassandra/pages/managing/operating/auto_repair.adoc

```diff
@@ -219,6 +219,10 @@ in nodetool repair. General advice is to keep this true.
 | repair_retry_backoff | 30s | Backoff time before retrying a repair session.
 | repair_task_min_duration | 5s | Minimum duration for the execution of a single repair task. This prevents the
 scheduler from overwhelming the node by scheduling too many repair tasks in a short period of time.
+| mixed_major_version_repair_enabled | false | Enable/Disable running repairs on the cluster when there are mixed.
+major versions detected, which usually occurs when the cluster is being upgraded. Repair is not supported in this scenario
```

Comment:
```
Repair is not supported in this scenario as data might be streamed from a node with a different version of Cassandra leading to compatibility issues. Once major versions are compatible for streaming data, this setting can be turned to true.
```

Could we just rephrase this as:

```
Repairs between nodes of different major versions is not something that is tested, so this may lead to data compatibility issues.  It is strongly discouraged to set this to true without doing extensive testing beforehand.
```